app/views/play/level/tome/SpellView.coffee

```diff
@@ -47,6 +48,8 @@ module.exports = class SpellView extends CocoView
     'tome:maximize-toggled': 'onMaximizeToggled'
     'script:state-changed': 'onScriptStateChange'
     'playback:ended-changed': 'onPlaybackEndedChanged'
+    'level:contact-button-pressed': 'saveSpade'
```

Comment:
For style consistency, we should probably call this `onContactButtonPressed` and then inside that method we can call the `saveSpade` method.